app/controllers/users_controller.rb

```diff
@@ -24,10 +24,9 @@ def new
   def edit; end
 
   def create
-    user_params = params.require(:user).permit(*WHITELISTED_ATTRIBUTES)
-    membership_params = user_params[:membership]
-    user_params = parse_roster_ids(user_params.except(:membership))
-    @user = User.new(user_params)
+    create_user_params = params.require(:user).permit(*WHITELISTED_ATTRIBUTES)
```

Comment:
Yeah. I started working on that but stopped because I was running into an issue when attempting to pull the new user's rosters in `user_params`, and in `update_membership`. I'll play with it though.